packages/bot-web-ui/src/stores/quick-strategy-store.js

```diff
@@ -170,6 +173,18 @@ export default class QuickStrategyStore {
         }
     }
 
+    @action.bound
+    async toggleInfoModal() {
```

Comment:
do we need async action?

Reply:
This function is similar to the `toggleStrategyModal` function on line 167, that's why I've followed a similar pattern here.

Reply:
Ok, We use async when we want to await for a response in toggleStrategyModal on line 169 we have await function (            await this.updateSymbolDropdown();) because of that we need async there

Reply:
Ahh, alright. I'll make this change, thank you! 😄